osgp/shared/shared/src/main/java/org/opensmartgridplatform/shared/security/RsaEncryptionService.java

```diff
@@ -142,7 +142,7 @@ private void initEncryption() {
      *             if the public key is not configured or if anything goes wrong
      *             while encrypting the given {@code input}.
      */
-    public byte[] encrypt(final byte[] input) {
+    public byte[] encrypt(final byte[] input) throws EncrypterException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'org.opensmartgridplatform.shared.exceptionhandling.EncrypterException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)